lib/data/network_services/network_service.dart

```diff
@@ -0,0 +1,56 @@
+import 'dart:convert';
+
+import 'package:http/http.dart';
+import 'package:spotify_prototype/data/models/track_model.dart';
+import 'package:spotify_prototype/data/network_services/mocked_network_service.dart';
+
+class NetworkServiceImpl extends NetworkService {
+  static const tokenValue =
```

Comment:
are u planning to implement token getting properly?

Reply:
Yes, I`m planning.